Client/Frontend/Home/ReaderPanel.swift

```diff
@@ -206,18 +227,96 @@ class ReadingListPanel: UITableViewController, HomePanel, SWTableViewCellDelegat
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        tableView.rowHeight = ReadingListPanelUX.RowHeight
+
+        tableView.rowHeight = ReadingListTableViewCellUX.RowHeight
         tableView.separatorInset = UIEdgeInsetsZero
         tableView.layoutMargins = UIEdgeInsetsZero
         tableView.separatorColor = AppConstants.SeparatorColor
         tableView.registerClass(ReadingListTableViewCell.self, forCellReuseIdentifier: "ReadingListTableViewCell")
 
         view.backgroundColor = AppConstants.PanelBackgroundColor
-    }
 
-    override func viewWillAppear(animated: Bool) {
         if let result = profile.readingList?.getAvailableRecords() where result.isSuccess {
             records = result.successValue
+
+            // If no records have been added yet, we display the empty state
+            if records?.count == 0 {
+                tableView.scrollEnabled = false
+
+                let overlayView = UIView(frame: tableView.bounds)
```

Comment:
tableView.bounds isn't calculated yet in viewDidLoad because a layout pass hasn't occurred yet. This will set the overlayView's frame to CGRectZero. I don't think that's the related to the issue you were having below with autolayout not working but just wanted to point it out.